sa/sa.go

```diff
@@ -709,7 +709,8 @@ func (ssa *SQLStorageAuthority) CountFQDNSets(ctx context.Context, req *sapb.Cou
 
 // FQDNSetTimestampsForWindow returns the issuance timestamps for each
 // certificate, issued for a set of domains, during a given window of time, in
-// ascending order.
+// ascending order. If no issuance timestamps are found, a NotFound error is
```

Comment:
It actually seems strange to treat "no issuance timestamps found" as an error condition. It's an expected outcome during normal operations, and can be expressed by returning a `*sapb.Timestamps` with zero entries. I think that would be a simpler and clearer way to express these semantics. You wouldn't need to add any special check for zero rows, and would not need to manually promote to a `berrors.NotFoundError`.